### DISCUSSION TITLE
Could a zip with xcframeworks be included on the GitHub releases page?

### DISCUSSION THREAD

**Discussion by blakeprobinson** [firebase/firebase-ios-sdk#11449] [General]
Under `Integrate without using Swift Package Manager`, the getting started with Firebase documentation has a [section](https://firebase.google.com/docs/ios/setup#frameworks) for getting started with frameworks where you can download a zip consisting of xcframeworks. 

Would it be possible to include that zip on the [releases](https://github.com/firebase/firebase-ios-sdk/releases) page in GitHub? 

For context, I was looking for that zip for quite a while because I missed it in the documentation and I didn't think it existed because it wasn't part of the releases page. 

Many thanks for considering this.

**Comment by paulb777** [comment]
It's the Firebase.zip file in the Assets section of each release. See https://github.com/firebase/firebase-ios-sdk/releases/tag/10.11.0

**Reply by blakeprobinson** [reply]
Thanks for letting me know. Not sure how I missed this.
